### ISSUE TITLE
[Centreon-open-tickets]: install plugin to manage ticketing from monitoring

### ISSUE THREAD

**Issue by joschi99** [joschi99/Osiris#69]
Centreon-open-tickets provide the functionality to different ticketing systems like GLPI < 9.1, OTRS or trough email. With this plugin you can handle tickets from monitoring.

**Comment by joschi99** [comment]
- centreon-open-ticket: version 1.2.0 (patched for centreon 2.8.x, see https://github.com/centreon/centreon-open-tickets/issues/34)
- centreon 2.8.1

Installation
```
git clone https://github.com/centreon/centreon-open-tickets.git
install.sh -u /etc/centreon
```
- Go to the web interface to finish installation (see https://documentation.centreon.com/docs/centreon-open-tickets/en/latest/installation/index.html#centreon-web-installation)
- Install the widget (see https://documentation.centreon.com/docs/centreon-open-tickets/en/latest/installation/index.html#centreon-open-tickets-widgets-installation)
- Configure the provider (see https://documentation.centreon.com/docs/centreon-open-tickets/en/latest/configuration_guide/index.html)

**Comment by joschi99** [comment]
Note: at the moment the plugin will not work with GLPI 9.1.1, because the webservice plugin for GLPI is not compatible with the latest GLPI 9.1.x version.
